app/src/main/java/com/github/se/eventradar/viewmodel/qrCode/ScanFriendQrViewModel.kt

```diff
@@ -121,16 +211,18 @@ constructor(
     return updateResult is Resource.Success
   }
 
-  fun resetNavigationEvent() {
-    _uiState.value = _uiState.value.copy(action = Action.None)
+  private fun changeAction(action: Action) {
```

Comment:
(small thing) I cannot highlight the lines above, but the variable "uiLength" should probably be either a constant at the start of the file or somewhere else if you use this threshold elsewhere in the app

Reply:
yes still waiting on Achille to confirm if he has got rid of adding extra information to encoding... will actually be deleted soon